ports/qcbor/vcpkg.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "qcbor",
+  "version": "1.5.3",
+  "description": "Comprehensive, powerful, commercial-quality CBOR encoder/ decoder that is still suited for small devices.",
+  "homepage": "https://github.com/laurencelundblade/QCBOR",
+  "license": "BSD-3-Clause",
+  "supports": "!windows | static",
```

No DLL export?
```suggestion
  "supports": "!windows | !(static & staticcrt)",
```
and
~~~cmake
if(VCPKG_TARGET_IS_WINDOWS)
    vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
endif()
~~~
in the portfile.

---

(Rectified suggestion.)